server/chat-commands/admin.ts

```diff
@@ -420,6 +420,57 @@ export const commands: Chat.ChatCommands = {
 		`If a [highlight] is specified, only highlights them if they have that term on their highlight list.`,
 	],
 
+	changeprivateuhtml: 'sendprivatehtml',
+	sendprivateuhtml: 'sendprivatehtml',
+	sendprivatehtml(target, room, user, connection, cmd) {
+		room = this.requireRoom();
+		this.checkCan('addhtml', null, room);
+
+		const {targetUser, rest} = this.requireUser(target);
+
+		if (targetUser.locked && !this.user.can('lock')) {
+			this.errorReply("This user is currently locked, so you cannot send them private HTML.");
+			return false;
```

Comment:
```suggestion
			throw new Chat.ErrorMessage("This user is currently locked, so you cannot send them private HTML.");
```
This seems more consistent